src/SDKs/RecoveryServices.Backup/Management.RecoveryServices.Backup/Generated/Models/PreValidateEnableBackupRequest.cs

```diff
@@ -74,6 +77,12 @@ public PreValidateEnableBackupRequest(string resourceType = default(string), str
         [JsonProperty(PropertyName = "resourceId")]
         public string ResourceId { get; set; }
 
+        /// <summary>
```

Comment:
Please bump the Nuget minor version (in .csproj and AssemblyInfo.cs) and update PackageReleaseNotes